pkg/rtc/transport.go

```diff
@@ -390,7 +390,7 @@ func NewPCTransport(params TransportParams) (*PCTransport, error) {
 		params:                   params,
 		debouncedNegotiate:       debounce.New(negotiationFrequency),
 		negotiationState:         NegotiationStateNone,
-		eventCh:                  make(chan event, 50),
+		eventCh:                  make(chan event, 100),
```

Comment:
Seeing a handful of logs of event queue getting full. Seems to happen mostly when some client has a ton of remote ICE candidates. Bumping this up to handle that.